source/blog/2017-10-16-fall-in-for-finagle.md

```diff
@@ -0,0 +1,144 @@
+regularly on a quarterly basis. You can start with a recap of what we were talking about [last time](https://finagle.github.io/blog/2017/07/12/summer-review/).
+
+### News & Notes
+
+------------
+
+- Finagle's 7th "birthday" (7th anniversary of the [first public
+commits](https://github.com/twitter/finagle/tree/e04e51645374f8d958d85de384142dd00f4b7574))
+is approaching. Stay tuned for a celebratory blog post!
+
+- We're retiring the [Finatra](https://github.com/twitter/finatra)-specific
+blog, formerly available
+[here](https://twitter.github.io/finatra/blog/archives/). This will happen with 
+our next release scheduled in the next few weeks. Unfortunately, we won't be 
+keeping any of the old posts but any new information will be shared on the 
+finagle blog here.
+
+---------
+
+#### Efficiency
+
+Metrics ‘verbosity’ level. Similar to logging levels in a logging API, we've
+introduced the concept of a verbosity level for metrics exported by Finagle
+([1](https://github.com/twitter/finagle/commit/6c666ab5c3363ae5bd22b0fbd96f33995fe36ac7),
+[2](https://github.com/twitter/util/commit/5e03745015693bc61fa2aace8da5eb452ff6e53d),
+[3](https://github.com/twitter/util/commit/d08661fe49377aafd8af60231d6112d546e55e01),
+[4](https://github.com/twitter/util/commit/1b3ce1eddbb2a9af9fa4f97f1fc1b20ac3ee6a40))
+with the ability for users to configure their service's verbosity and to
+define a verbosity when creating a new metric.
+
+For more information see the [Util](https://twitter.github.io/util) User's Guide section on [`Leveraging
+Verbosity
+Levels`](https://twitter.github.io/util/guide/util-stats/user_guide.html#leveraging-verbosity-levels).
+
+Our internal roll-out of our new load balancer, [Deterministic
+Aperture](https://github.com/twitter/finagle/blob/finagle-6.45.0/finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/aperture/DeterministicOrdering.scala) continues and we’re excited about bringing this to the wider community soon!
+
+### Resiliency
+
+Transparently replacing HTTP/1.1 usage with HTTP/2 is continues internally. HTTP/2 gives you the resource reductions (a single multiplexed connection) and resiliency features (fast rolling restarts without a success rate drop) that services are already accustomed to with Mux.
+
+### Operability
+
+We’ll be open-sourcing the work done by our summer intern [@McKardah](https://twitter.com/McKardah) to wire
+Twitter Futures into [IDEA’s async
+stacktraces](https://blog.jetbrains.com/idea/2017/02/intellij-idea-2017-1-eap-extends-debugger-with-async-stacktraces/).
+
+We’ve completed the internal rollout of [edge-triggered native
+transport](https://github.com/twitter/finagle/search?utf8=%E2%9C%93&q=nativeEpoll&type=) using Netty’s JNI edge-triggered transport, instead of using the JDK’s NIO and are working to make it available to configure externally.
```

Comment:
its already available externally. we should point folks at the tunable/flag `com.twitter.finagle.netty4.UseNativeEpollV2` and recommend that they try using it.

also, i think this should move to the Efficiency section.